backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/transfer/HUTransformService.java

```diff
@@ -1161,6 +1161,26 @@ public I_M_HU huToNewSingleCU(@NonNull final HUsToNewCUsRequest newCURequest)
 		return result.build();
 	}
 
+	@NonNull
```

Comment:
```suggestion
	/**
	  * If the given HU is aggregated, split out one TU and assign the given QR-code to it. Otherwise do nothing and jsut return the given {@code huId}.
	  */
	@NonNull
```